picard/ui/options/plugins.py

```diff
@@ -599,16 +601,32 @@ def refresh_details(self, item):
         if plugin.description:
             text.append(plugin.description + "<hr width='90%'/>")
         infos = [
-            (_("Name"), plugin.name),
-            (_("Authors"), plugin.author),
+            (_("Name"), escape(plugin.name)),
+            (_("Authors"), self.link_authors(plugin.author)),
             (_("License"), plugin.license),
-            (_("Files"), plugin.files_list),
+            (_("Files"), escape(plugin.files_list)),
         ]
         for label, value in infos:
             if value:
                 text.append("<b>{0}:</b> {1}".format(label, value))
         self.ui.details.setText("<p>{0}</p>".format("<br/>\n".join(text)))
 
+    @staticmethod
+    def link_authors(authors):
+        formatted_authors = []
+        re_author = re.compile(r"(?P<author>.*?)\s*<(?P<email>.*?)>")
+        for author in authors.split(','):
+            match = re_author.match(author.strip())
```

Comment:
```suggestion
            match = re_author.fullmatch(author.strip())
```

Else it will match stuff like `some<thing>else` (I think).